tests/modules/nf-core/pilon/test.yml

```diff
@@ -0,0 +1,8 @@
+- name: pilon test_pilon
+  command: nextflow run ./tests/modules/nf-core/pilon -entry test_pilon -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/pilon/nextflow.config
+  tags:
+    - pilon
+  files:
+    - path: output/pilon/test.fasta
+      md5sum: 2e881994820a5a641da9ea594ab4958f
+    - path: output/pilon/versions.yml
```

Comment:
shouldnt the version also be reproducible, i.e. the file should also have a stable md5sum??

Reply:
I just checked in other test.yml files and the version.yml is usually not associated to a md5sum (I don't know why),, but I think that's ok.

Example with DeepVariant : https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/deepvariant/test.yml
Example with Hifiasm : https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/hifiasm/test.yml